lib/VerifiableCredentialStore.js

```diff
@@ -10,7 +10,11 @@ export default class VerifiableCredentialStore {
   constructor({edv, invocationSigner}) {
     this.edv = edv;
     this.invocationSigner = invocationSigner;
-    this.edv.ensureIndex({attribute: ['meta.issuer', 'content.type']});
+    this.edv.ensureIndex({attribute: [
+      'meta.issuer',
+      'meta.listDisplay',
```

Comment:
What does `listDisplay` mean? This seems like it is very UI-component centric. Is there another name we could use that would be more semantic? This property indicates what about the VC? Imagine we aren't going to display VCs in a list but some other way in the future -- but we still need to use this property and it's now "historically weird" to people. What name could we use to avoid that?

Reply:
@dmitrizagidulin I think this was your suggestion. Do you have any comments about this?

Reply:
@dlongley - it is in fact a UI-component centric name. If we want it to be future-ish-proof, let's call it `displayable` then.